doc/deprecation.md

```diff
@@ -498,6 +498,31 @@ def example_function(n_clusters=10, another=42, k="deprecated"):
 example_function(k=10)
 ```
 
+## Deprecation warning with logger
+
+`deprecation_warning(telemetry, message)` is designed to facilitate the logging of deprecated features in your codebase. It throws warning messages and log
+the information about the message, the current package name and version.
+
+We need to provide telemetry instance and the message string:
+
+- telemetry, [Telemetry class](https://ploomber-core--65.org.readthedocs.build/en/65/telemetry.html)
```

Comment:
use relative links, this is a link to the PR documentation.

Reply:
Done!